package.json

```diff
@@ -113,6 +113,7 @@
     "@typescript-eslint/parser": "5.48.0",
```

Comment:
could you do me a random favor and update the eslint:strict command in this file to `"eslint:strict": "STRICT=1 eslint '*.{js,ts,tsx}' 'src/**/*.ts?(x)' 'cypress/**/*.ts' 'src/gql/**/*.graphql'"`, I couldn't do it in my lint PR because it errors when there are no GraphQL files 🥹